api/infrastructure/parallelcluster-api.yaml

```diff
@@ -501,6 +501,37 @@ Resources:
                   aws:RequestedRegion:
                     - !Ref Region
             Sid: EC2Write
+          - Action:
+              - autoscaling:DescribeAutoScalingGroups
+              - autoscaling:DescribeScalingActivities
+              - autoscaling:DescribeAutoScalingInstances
+              - autoscaling:DescribeScheduledActions
+            Resource: '*'
+            Effect: Allow
+            Condition: !If
+              - IsMultiRegion
+              - !Ref AWS::NoValue
+              - StringEquals:
+                  aws:RequestedRegion:
+                    - !Ref Region
+            Sid: AutoScalingList
+          - Action:
+              - autoscaling:CreateAutoScalingGroup
+              - autoscaling:DeleteAutoScalingGroup
+              - autoscaling:UpdateAutoScalingGroup
+              - autoscaling:SuspendProcesses
+              - autoscaling:TerminateInstanceInAutoScalingGroup
+              - autoscaling:ResumeProcesses
+              - autoscaling:SetDesiredCapacity
+            Resource: '*'
```

Comment:
Can we restrict this resource and the above one to only work with ASGs that are created as part of ParallelCluster stacks? a ParallelCluster prefix in the ASG name can be a way to filter them

Reply:
we don't have parallelcluster prefix in ASG name now, for example, 
`arn:aws:autoscaling:eu-west-3:109993544819:autoScalingGroup:ae217271-ec3f-495f-b259-91db02ea9bec:autoScalingGroupName/integ-tests-f9c8ssa4wvv8udci-p3-HeadNode-ZH9RRWSB8X1P`
it is composed of {cluster_name}-HeadNode-{random_string}, shall I filter it with `arn:${AWS::Partition}:autoscaling:${Region}:${AWS::AccountId}:autoScalingGroup:*/*-HeadNode-*` or rename the ASG name with parallelcluster prefix?

Reply:
my personal preference is to rename the ASG with a parallelcluster prefix, since that would seem to better ensure that the policy can only be used for pcluster-specific resources.

Reply:
done